pkg/controller/manilacsi/manilacsi_controller.go

```diff
@@ -117,8 +118,14 @@ func (r *ReconcileManilaCSI) Reconcile(request reconcile.Request) (reconcile.Res
 func (r *ReconcileManilaCSI) handleManilaCSIDeployment(instance *manilacsiv1alpha1.ManilaCSI, reqLogger logr.Logger) (reconcile.Result, error) {
 	reqLogger.Info("Reconciling ManilaCSI Deployment Objects")
 
+	// Credentials Secret
+	err := r.handleManilaCrednetials(instance, reqLogger)
```

Comment:
What does this function do? `handle` is not very talkative.

If it ensures that a secret is in place, you could call it `r.ensureManilaSecret`. If it reconciles the Manila secret with the cloud credentials, you could call it `r.reconcileManilaCredentials`.

(I am sure you have an idea for a better name than those two I'm suggesting)